cmd/feed-ingress/main.go

```diff
@@ -292,6 +300,8 @@ func createIngressUpdaters() ([]controller.Updater, error) {
 			BackendHealthcheckInterval: gorbBackendHealthcheckInterval,
 			BackendHealthcheckType:     gorbBackendHealthcheckType,
 			InterfaceProcFsPath:        gorbInterfaceProcFsPath,
+			HTTPClientTimeout:          gorbHTTPClientTimeout,
+			HTTPClientMaxRetries:       gorbHTTPClientMaxRetries,
```

Comment:
I don't think we need to make the timeouts configurable. The ELB time outs are not configurable either.

Reply:
have removed the flags from feed-ingress and set the values in gorb